finance/community-updates/2021-11.txt

```diff
@@ -0,0 +1,100 @@
+proposal to the NASA ROSES-2020 call, section E.7 (Support for Open
+Source Tools, Frameworks, and Libraries). The proposal included a mix of
+Project-wide infrastructure improvements, targeted work on specific
+areas of the code that are in need of support, enhancements to the our
+educational materials, and better support for affiliated packages. The
+full text (with some personal details redacted) is available at [1].
+
+The proposal was successful, and has provides a further $634,590 to
+support the fantastic work being undertaken on Astropy. The Finance
+Committee will arrange processes for disbursing the funds which are
+consistent with the goals of the proposal and based on the model we've
+adopted for assigning Moore funding over the past couple of years.
+
+This is a huge milestone for the project: it shows that we've
+successfully used the Moore funding to build a solid set of processes
+and a governance structure which enables us to obtain funding from more
+“traditional” academic sources. This is a great step towards ensuring
+the financial viability of the project into the future.
+
+[1] https://drive.google.com/file/d/18JsTRahNWnvign6ao6BP_OkINHnP8g3n/view
+
+# Gemini contract
+
+The Science User Support Department at Gemini Observatory has funded
+Simon Conseil as an Astropy contractor to provide maintenance
+and development that benefits both the Astropy and DRAGONS projects on a
+part-time basis (averaging 5 hours per week over a period of a year or
+two). The details of the work are described in the statement of work [2].
+
+Having major facilities like Gemini invest in Astropy is another sign of
+both the critical role that the project plays in astronomical
+infrastructure worldwide, and of the increasing level of trust displayed
+in the project's responsible and sustainable governance.
+
+[2] https://docs.google.com/document/d/1sKiFr5ypS3prkEqEu3s5nEejs0bD93cF7ihXSz3RdZM/edit
+
+# Distribution of Moore funds
+
+We organized a process to select and distribute funds from the Moore
+grant to a variety of projects, suggested and selected by the project
+membership. We've announced results in detail in the past [3] so we
+won't reiterate them in detail here, but we're delighted to report that
+most many of the projects are now well underway.
+
+We except to organize similar processes to distribute Moore and NASA
+funding in future years. We welcome your feedback on how to continue to
+improve our approach to make this as transparent and effective as
+possible.
+
+[3] https://groups.google.com/g/astropy-dev/c/c1M2F7rZoF4/m/nbNdg1OsAAAJ
+
+# Meetings, workshops, and other travel
+
+We've supported project members to participate in and represent Astropy
+at a number of conferences and workshops over the last year.
+These include:
+
+- Three project members were funded to (virtually) attended ADASS XXX,
+  where Brigitta Sipocz spoke on “astroquery: a community driven
```

Comment:
```suggestion
- Three project members were funded to (virtually) attended ADASS XXXI,
  where Brigitta Sipocz spoke on “astroquery: a community driven
```

Reply:
No, it really was “XXX” — our last mail went out before that meeting. I'm not aware of us funding anybody to go to ADASS XXXI — if we did, we don't seem to have written an issue about it.